pytest_mccabe.py

```diff
@@ -41,7 +41,13 @@ def pytest_collect_file(path, parent):
     if config.option.mccabe and path.ext == '.py':
         complexity = config._mccabe_complexities(path)
         if complexity != 0:
-            return McCabeItem(path, parent, complexity)
+            if hasattr(McCabeItem, "from_parent"):
```

Comment:
I don't think there's a good reason to still support pytest < 5.4.0. IMHO it'd be fine to drop it and only support newer versions.

Reply:
Cool - I removed that part :+1: